pkg/operator/controllers/muo/deploy_test.go

```diff
@@ -233,81 +234,75 @@ func TestDeployConfig(t *testing.T) {
 	tests := []struct {
 		name             string
 		deploymentConfig *config.MUODeploymentConfig
-		expected         []string
+		expected         string
 	}{
 		{
 			name:             "local",
 			deploymentConfig: &config.MUODeploymentConfig{EnableConnected: false},
-			expected: []string{
```

Comment:
this looks ugly :( do we have a better way of doing this?

Reply:
We could read it from a file, using embed. But really here I just used what was done before just removing the useless array